projects/ngrx.io/content/guide/store/index.md

```diff
@@ -9,6 +9,16 @@ Store is RxJS powered state management for Angular applications, inspired by Red
 - [Selectors](guide/store/selectors) are pure functions used to select, derive and compose pieces of state.
 - State accessed with the `Store`, an observable of state and an observer of actions.
 
+## Installation
+
+```sh
+$ npm install @ngrx/store --save
```

Comment:
Remove the `$`  an leading space so it can be used as copied.

Reply:
done